scripts/get_message_templates.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+path="$1"
+function getMessageTemplates() {
+  grep -R "\\.withMessageTemplate" --exclude="*Test*" --exclude="*.class" "$path" | awk -F'"' '{sub(/[ \t]+$/, "", $2); sub(/[ \t]+$/, "", $3); print $2, $3, $4}' | sort -g | uniq -u | sed "s/,/=/g" > dictionary.properties
```

Comment:
Instead of excluding `.class` files, it would be better to only include `.java` files.